pkgs/os-specific/darwin/apple-sdk-11.0/libpm.nix

```
@@ -0,0 +1,23 @@
+{ stdenvNoCC, buildPackages, MacOSX-SDK }:
+
+stdenvNoCC.mkDerivation {
+  pname = "libpm";
+  version = MacOSX-SDK.version;
+
+  dontUnpack = true;
+  dontBuild = true;
+
+  nativeBuildInputs = [ buildPackages.darwin.checkReexportsHook ];
```

Isn't the explicit `buildPackages` reference redundant here due to splicing?

```suggestion
  nativeBuildInputs = [ darwin.checkReexportsHook ];
```

---

Removed.